tests/testthat/setup.R

```diff
@@ -20,7 +20,9 @@ if (nzchar(Sys.getenv("MEETUPR_PWD"))) {
   meetup_auth(token = temptoken)
 
 } else {
-  Sys.setenv("MEETUPR_TESTING" = TRUE)
+  if (!identical(Sys.getenv("NOT_CRAN"), "true")) {
```

Comment:
on CRAN it's like on CI (apart from the workflow in with-auth.yml) we only want to use the fixtures.

Reply:
I see where you're going, but the double-negative of !NOT-THING isn't ideal. Can we word it as a positive-case, perhaps set "USE_CRAN: true" in the CI config and then it doesn't have to be explicitly disabled by users?

Reply:
ouch I think this might be where the problem comes from (although I'd like to keep the idea of this code I took from `testthat:::on_cran()`)

Reply:
You shouldn't need to worry about `NOT_CRAN` as it is set by devtools (I just re-read `?testthat::skip_on_cran`)

Reply:
Fair enough. Double-negatives are annoying though, though I doubt `testthat` would change it given the sheer number of downstream packages that would break ;)

I'm no fan of calling package internals, but would it be better to call

```r
if (testthat:::on_cran) {
```

To make the double-negative go away *and* also not duplicate their code into meetupr?

Reply:
well as it's not exported I don't think we can. :grimacing:

Reply:
Fair enough. My own package-writing skills are less polished, I wasn't sure if we could access it during tests (obviously it's not relevant to a production install). Duplication it is - although we could copy the `on_cran()` function to `internals.R` and then add a link to the source. But I'm nitpicking here, for sure.